docs/_includes/tk-3dsmax

```diff
@@ -1 +1 @@
-2018+
\ No newline at end of file
+2023
```

Comment:
Do we only support one version? What happened to the "current + 2 previous ones" rule? Does that not apply here?